src/twisted/mail/alias.py

```
@@ -723,18 +726,18 @@ def __init__(self, items, *args):
         self.aliases = []
         while items:
             addr = items.pop().strip()
-            if addr.startswith(':'):
+            if addr.startswith(b':'):
                 try:
-                    f = open(addr[1:])
-                except:
+                    f = open(addr[1:], "rb")
+                except Exception:
```

Is it appropriate to swallow the error like this without propagating the error to the caller?

I think either the `except:` clause should be narrowed to only "invalid filenames", or the log message should include the exception that was raised. (Perhaps both: narrow to `OSError` and `ValueError`, and also report the exception?)

---

This can be done in a different ticket, it is not related to porting to Python 3.